mne/forward/_make_forward.py

```diff
@@ -323,7 +323,7 @@ def _prep_meg_channels(info, accurate=True, exclude=(), ignore_ref=False,
 
 
 @verbose
-def _prep_eeg_channels(info, exclude='bads', verbose=None):
+def _prep_eeg_channels(info, exclude=(), verbose=None):
```

Comment:
That was only there for backward compatibility before. It should either be `exclude='bads'` or `exclude=[]` (or better, `exclude=()` because it's immutable)